src/frontend/komponenter/Fagsak/Vilkår/BehandleVilkårSkjema.tsx

```diff
@@ -82,6 +82,21 @@ const BehandlingVilkårSkjema: React.FunctionComponent<IBehandlingVilkårSkjema>
                     });
                 }}
             />
+
+            <br/>
+            
+            <TextareaControlled 
+                label={'Begrunnelse'}
+                maxLength={0}
+                defaultValue={context.begrunnelse}
```

Comment:
I think we should use value from the context and not defaultValue. This way it is easier to load begrunnelse from vedtak also. 

Because I would like us to always try to load the state from the case. Like I did with beregning and hjemler. So that when a new case worker goes into the case the original begrunnelse is presented.

Reply:
Ok, actually I don't know what the defaultValue is intended for when value is present. Should I write:
defaultValue= {''}
value= {context.begrunnelse}

Reply:
Only value should be fine.

Reply:
defaultValue is mandatory

Reply:
Ok, then I think we should set '' as default value and overwrite it with value={context.begrunnelse}.

Reply:
According to https://design.nav.no/components/textarea/technical defaultValue is not required.